### PR TITLE
Toggle

### PR DESCRIPTION
**Pull request by antictor**
toggle
 A simple toggle component for use in forms or as a ui element to control something.

- [x] Documentation
- [x] Sass styles
- [x] Vue JS Component
- [x] Storybook example of component
- [x] Jest test

### CODE REVIEW

File: packages/documentation/library/pages/develop/components/toggle.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+A toggle is a simple component that acts like a switch, with a tue or false state, though when used in HTML forms this changes and results in the inclusion or exclusion of a value.
+
+## Usage
+
+The toggle component can be used as a Vue/React component anywhere in a page or within a html form.
+
+### HTML
+
+When used in a regular web page the toggle component can only be used in a HTML form and uses markup identical to a checkbox but with classes to indicate the component should be progressivey enhanced to appear like a toggle.
```
Comment (EdPoole) [inline packages/documentation/library/pages/develop/components/toggle.md]:
```suggestion
The HTML version of the component uses a checkbox input to toggle its state. The `.rn-toggle` classes progressively enhance the look of the checkbox into that of the toggle switch.
```

File: packages/documentation/library/pages/develop/components/toggle.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    type="checkbox"
+    value="yes"
+  />
+  <span class="rn-toggle__label-inner">Toggle title</span>
+</label>
+```
+
+### Vue JS
+
+A Vue JS toggle can be used either in a form or on it's own, maybe to control part of a page.
```
Comment (EdPoole) [inline packages/documentation/library/pages/develop/components/toggle.md]:
```suggestion
A Vue JS toggle can be used either in a form or on its own.
```

File: packages/css-framework/src/components/_toggle.scss [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  display: inline-block;
+  position: relative;
+  height: 10px;
+  width: 21px;
+  background: color(neutral, 50);
+  border: 1px solid color(neutral, 300);
+  border-radius: 5px;
+}
+
+.rn-toggle.rn-toggle.is-disabled .rn-toggle__track {
```
Comment (EdPoole) [inline packages/css-framework/src/components/_toggle.scss]: Is the over-specification of `.rn-toggle` here intentional?

Comment (antictor) [reply packages/css-framework/src/components/_toggle.scss]: Nope thats a typo

**Pull request closed without merging**